aspen/wsgi.py

```diff
@@ -18,3 +18,4 @@
 from aspen.website import Website
 
 website = Website([])
+application = website  # many servers look for this name
```

Comment:
Nevermind. I was just trying to make it easier for people who don't know what a WSGI server is and will think that this can be served by Apache.